### PR TITLE
Feature/mcbobby/create learn more page

### PR DESCRIPTION
**Pull request by mcbobby7**
add learn more landing page using faker.js

### CODE REVIEW

File: src/components/learnMore/constants.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */
+/* eslint-disable sort-keys */
```
Comment (EngrEric) [inline src/components/learnMore/constants.js]: remove the eslint disble and write your code to conform

File: src/components/router/components/ScrollToTop.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    componentDidUpdate(prevProps) {
+        if (this.props.location.pathname !== prevProps.location.pathname) {
+            window.scrollTo(0, 0);
+        }
+    }
+    render() {
+        return this.props.children;
+    }
+}
+
```
Comment (EngrEric) [inline src/components/router/components/ScrollToTop.js]: this changes here is already in develop branch.
remove it

File: src/components/userProfile/components/UserProfileModal.js [hunk 3]
```diff
@@ -68,7 +77,7 @@ const UserProfileModal = Form.create({ name: FORM_IN_MODAL })(
                 <Modal
                     centered={true}
                     visible={visible}
-                    width={MODAL_WIDTH}
+                    // width={MODAL_WIDTH}
```
Comment (EngrEric) [inline src/components/userProfile/components/UserProfileModal.js]: remove unused code

File: src/components/router/components/ScrollToTop.js (hunk 2 above)
Comment (mcbobby7) [reply src/components/router/components/ScrollToTop.js]: Yea i pulled it from develop

**Review by EngrEric — APPROVED** [review]
nice, looks good to me

**Pull request merged**